sceptre/cli.py

```diff
@@ -574,6 +576,170 @@ def get_stack_policy(ctx, environment, stack):
     write(response.get('StackPolicyBody', {}))
 
 
+@cli.group(name="init")
+def init():
+    """
+    Commands for initialising Sceptre projects.
+    """
+    pass
+
+
+@init.command("env")
+@click.argument('env')
+@catch_exceptions
+@click.pass_context
+def init_environment(ctx, env):
+    """
+    Initialises an environment in a project.
+
+    Creates ENVIRONMENT folder in the project.
+    """
+    cwd = ctx.obj["sceptre_dir"]
+    for item in os.listdir(cwd):
+        # If already a config folder create a sub environment
+        if os.path.isdir(item) and item == "config":
+            config_dir = os.path.join(os.getcwd(), "config")
+            _create_new_environment(config_dir, env)
+
+
+@init.command("project")
+@catch_exceptions
+@click.argument('project_name')
+@click.pass_context
+def init_project(ctx, project_name):
+    """
+    Initialises a new project.
+
+    Creates PROJECT_NAME project folder.
+    """
+    cwd = os.getcwd()
+    sceptre_folders = {"config", "templates"}
```

Comment:
Does sceptre_folders need to be explicitly declared as a variable? It looks like it's only used in one place.

Reply:
Technically not - I'm being overly verbose just to help the readability of the code. Can take it out if you think its too much.

Reply:
I'm torn on whether it makes it more readable (since what it's referring to is so simple), I'll defer to your judgement.